chore(connectors): update docs for rabbitMQ, split routing data (#1643)
camunda/camunda-docs: docs/components/connectors/out-of-the-box-connectors/rabbitmq.md

## Routing data

In the **Routing** section you must set the routing data attributes:
For a **URI** type connection, the required fields are `exchange` and `routingKey`.
For a **Credentials** type connection, the required fields are `exchange`, `routingKey`, `virtualHost`, `hostName`, and `port`.

> - Go to [RabbitMQ Exchanges, routing keys and bindings documentation](https://www.cloudamqp.com/blog/part4-rabbitmq-for-beginners-exchanges-routing-keys-bindings.html) for learn more about RabbitMQ exchanges and routing keys.
> - Go to [RabbitMQ virtual host documentation](https://www.rabbitmq.com/vhosts.html) for learn more about RabbitMQ virtual hosts.
> - Go to [RabbitMQ networking documentation](https://www.rabbitmq.com/networking.html) for learn more about configuration RabbitMQ host and port.
> 
> ![connectors-rabbitmq-routing](../img/connectors-rabbitmq-routing-uri.png)
> ![connectors-rabbitmq-routing](../img/connectors-rabbitmq-routing-cred.png)

Reviewer: A more concise language here might be more readable. What do you think about this?

```suggestion
Please refer to RabbitMQ documentation to learn about routing attributes:
- [Exchanges, routing keys and bindings](https://www.cloudamqp.com/blog/part4-rabbitmq-for-beginners-exchanges-routing-keys-bindings.html)
- [Virtual hosts](https://www.rabbitmq.com/vhosts.html)
- [Networking, host and port configuration](https://www.rabbitmq.com/networking.html)
```

Reviewer 2: I really like this suggestion! I would just adjust to:

~~~Please~~~ Refer to RabbitMQ documentation to learn about routing attributes:
- [Exchanges, routing keys**,** and bindings](https://www.cloudamqp.com/blog/part4-rabbitmq-for-beginners-exchanges-routing-keys-bindings.html)
- [Virtual hosts](https://www.rabbitmq.com/vhosts.html)
- [Networking, host**,** and port configuration](https://www.rabbitmq.com/networking.html)

Reviewer 2: Adjusting this in my technical review/latest commit. 👍
